airbyte-config-oss/init-oss/src/main/resources/seed/source_specs.yaml

```diff
@@ -15257,7 +15257,7 @@
     supportsNormalization: false
     supportsDBT: false
     supported_destination_sync_modes: []
-- dockerImage: "airbyte/source-stripe:3.4.3"
+- dockerImage: "airbyte/source-stripe:3.5.0"
   spec:
     documentationUrl: "https://docs.airbyte.com/integrations/sources/stripe"
     connectionSpecification:
```

Comment:
Can you delete this file? We're no longer using this file on master and instead are relying on metadata.yaml moving forward.